ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/vlan-interfaces.cfg

```diff
@@ -195,6 +195,24 @@ interface Vlan333
    vrrp 3 ipv4 100.64.0.1
    vrrp 3 ipv4 version 3
 !
+interface Vlan334
+   description v6 attached host exports
+   ipv6 enable
+   ipv6 address 2a00:1::1/64
+   ipv6 attached-host route export 19
+!
+interface Vlan335
+   description v6 attached host exports
+   ipv6 enable
+   ipv6 address 2a00:2::1/64
+   ipv6 attached-host route export prefix-length 64
+!
+interface Vlan336
+   description v6 attached host exports
+   ipv6 enable
+   ipv6 address 2a00:3::1/64
+   ipv6 attached-host route export 18 prefix-length 64
```

Comment:
This CLI line is rendered _above_ `ipv6 enable` on EOS. Please move it in the template.

Reply:
Fixed, thanks!